game-of-life.js

```diff
@@ -0,0 +1,249 @@
+let matrix; 
```

Comment:
Let's think that the game is like a movie and you have to calculate the next frame using the current frame. This means you need to have two matrces: `currentFrame`, `nextFrame`.